.tool-versions

```diff
@@ -1,2 +1,2 @@
-nodejs 14.17.3
+nodejs 22.11.0
```

Comment:
ok to update nodejs, or do you want to hold off on this?

Reply:
We can bump it :+1: